container-engine-lib/lib/channel_writer/channel_writer.go

```diff
@@ -0,0 +1,17 @@
+package channel_writer
+
+// Enables extracting output that is being written to an io.Writer via a channel
+type ChannelWriter struct {
+	channel chan string
+}
+
+func NewChannelWriter(channel chan string) *ChannelWriter {
```

Comment:
I'm surprised there isn't anything for this in go's std lib... But I couldn't find anything

Reply:
rightttt